CONTRIBUTIONS.md

```diff
@@ -0,0 +1,59 @@
+## Guidelines for Pull Requests and Releases (Web3 1.x)
+
+This document provides some ground rules for contributors (including the maintainer(s) of
+the project) about how to make, review and publish changes to 1.x. The most basic requirement is
+that **Web3 not break**.
+
+### Pull Requests for substantive changes (e.g. everything except comments and docs)
+
+1. Any PR that introduces a logic change should include tests. (In many cases, the tests will take
+more time to write than the actual code).
+
+2. All PRs should sit for 72 hours with the `pleasereview` tag in order to garner feedback.
+
+3. No PR should be merged until it has been reviewed, passes CI, and all reviews' comments are
+addressed.
+
+4. PRs should have a narrow, well-defined focus and make the smallest set of changes possible to achieve their goal. They should include a clear description in the opening comment.
```

Comment:
@cgewecke I think we should use a consistent keyword. I means the PR should be well-defined and `consistent`.

Reply:
@princesinha19 

To clarify, could you give an example of something that was not consistent? Are you concerned about the word 'narrow' and believe that a PR with several inter-connected goals is consistent and therefore acceptable?

Reply:
@cgewecke In some PR, people try to create unnecessary typings for example also they do not write code in the same way. Like if we use camelcase for variables then one shouldn't use a snake-case or other cases. Also, One should not use Array<T> for typings if, in the whole module, we have use T[ ], this will violate consistency.

Reply:
@princesinha19. Ah yes that's a great idea, thanks for that suggestion.

Reply:
@princesinha19 Added a note about this in 89704e19f986c36173d7dc3989747afce849236b. It says that people should follow the existing conventions and preserve stylistic consistency.

Reply:
Great Thanks.

Reply:
@princesinha19 Thanks for your input!